### PR TITLE
adding simple demo

### PR DESCRIPTION
**Pull request by jab0707**
added the simple demo file. Some of the comments make references to other examples which need updated

### CODE REVIEW

File: demos/basic_uq_example.py [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+#  np.linalg.norm( pce.coefficients - pce2.coefficients )
+
+# # Postprocess PCE: mean, stdev, sensitivities, quantiles
+mean = pce.mean()
+stdev = pce.stdev()
+
+# Power set of [0, 1, ..., dimension-1]
+variable_interactions = list(chain.from_iterable(combinations(range(dimension), r) for r in range(1, dimension+1)))
+
+# "Total sensitivity" is a non-partitive relative sensitivity measure per parameter.
```
Comment (akilnarayan) [inline demos/basic_uq_example.py:134]: Maybe remove total sensitivity since you don't plot it?

File: demos/basic_uq_example.py [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+total_sensitivity = pce.total_sensitivity()
+
+# "Global sensitivity" is a partitive relative sensitivity measure per set of parameters.
+global_sensitivity = pce.global_sensitivity(variable_interactions)
+
+Q = 3  # Number of quantile bands to plot
```
Comment (akilnarayan) [inline demos/basic_uq_example.py:140]: Quantiles don't appear to be used. So remove them?

File: demos/basic_uq_example.py [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+q_lower = np.arange(dq, 0.5-1e-7, dq)[::-1]
+q_upper = np.arange(0.5 + dq, 1.0-1e-7, dq)
+quantile_levels = np.append(np.concatenate((q_lower, q_upper)), 0.5)
+
+quantiles = pce.quantile(quantile_levels, M=int(2e3))
+median = quantiles[-1, :]
+
+
+# # Visualization
+M = 1000  # Generate MC samples
```
Comment (akilnarayan) [inline demos/basic_uq_example.py:151]: The variable `M` doesn't seem to play a role here, meaning that you only use the first `V` samples and ignore all the rest. I'd just replace all `M` with `V` (which you set to 100)

**Review by akilnarayan — COMMENTED** [review]
I have some comments for simplifying the demo. If you think these are not applicable then let me know and I'll approve

**Comment by dcwhite** [comment]
I support the simplification of all included demos :)

**Comment by jab0707** [comment]
@akilnarayan I responded to your comments

**Review by akilnarayan — APPROVED** [review]

**Pull request merged**